e2e/openssl/openssl_test.go

```diff
@@ -43,9 +44,17 @@ var (
 func TestOpenSSL(t *testing.T) {
 	ct := contrasttest.New(t, imageReplacementsFile, namespaceFile, skipUndeploy)
 
-	resources := kuberesource.OpenSSL()
+	// TODO(msanft): Make this configurable
+	platform := platforms.AKSCloudHypervisorSNP
+
+	runtimeHandler, err := manifest.DefaultPlatformHandler(platform)
+	require.NoError(t, err)
 
+	resources := kuberesource.OpenSSL()
 	coordinator := kuberesource.CoordinatorBundle()
+
+	resources = kuberesource.PatchRuntimeHandlers(resources, runtimeHandler)
+
```

Comment:
The line has to be moved below `resources = append(resources, coordinator...)`. AFAICT this is why CI fails.